registry/client/repository.go

```diff
@@ -377,11 +377,7 @@ func (bs *blobs) Stat(ctx context.Context, dgst digest.Digest) (distribution.Des
 }
 
 func (bs *blobs) Get(ctx context.Context, dgst digest.Digest) ([]byte, error) {
-	desc, err := bs.Stat(ctx, dgst)
```

Comment:
This performs the function of mapping from local digest to canonical digest. I am not sure if this is necessary here, since the `Open` call should perform the same function.